src/rebase.c

```diff
@@ -967,6 +967,13 @@ static int rebase_commit__create(
 		goto done;
 	}
 
+	if ((!message || strcmp(message, git_commit_message(current_commit)) == 0) &&
+	    (!message_encoding || strcmp(message_encoding, git_commit_message_encoding(current_commit)) == 0) &&
```

Comment:
Yes to author, I think...  I expect most people will pass `NULL` but I think that if I wanted to force a new author I would be disappointed here...

No to committer, though.  Users would have to go get the committer information from the old commit to make this work, which seems like a terrible burden.